gulpfile.babel.js

```diff
@@ -275,11 +275,11 @@ gulp.task('default', ['clean'], cb => {
 // Build production files and microsite
 gulp.task('all', ['clean'], cb => {
   runSequence(
-    ['default', 'styletemplates'],
-    ['styles:gen'],
-    ['lint', 'scripts', 'assets', 'demos', 'pages',
-     'templates', 'images', 'styles-grid', 'metadata'],
+    ['styles:gen', 'styles-grid', 'styletemplates'],
```

Comment:
It might have worked in local tests by accident. I have run into that. But – again IIRC – `styletemplates` generates the templated version that `style:gen` uses to generate all combinations.